src/store/metadata/stub/content/editions/annual-personal-wellbeing-estimates_2023-09 .json

```diff
@@ -0,0 +1,122 @@
+{
+    "@context": "https://staging.idpd.uk/#ns",
+    "@id": "https://staging.idpd.uk/datasets/annual-personal-wellbeing-estimates/editions",
+    "@type": "hydra:Collection",
+    "title": "Annual personal wellbeing estimates",
+    "editions": [
+      {
+        "@id": "https://staging.idpd.uk/datasets/annual-personal-wellbeing-estimates/editions/2023-09",
+        "@type": "dcat:Dataset",
+        "in_series": "https://staging.idpd.uk/datasets/annual-personal-wellbeing-estimates",
+        "identifier": "2023-09",
+        "title": "Annual personal wellbeing estimates",
+        "summary": "Estimates of life satisfaction, feeling that the things done in life are worthwhile, happiness and anxiety for adults aged 16 years and over, UK, 04/2011-03/2012 to 04/2022-03/2023.",
+        "description": "This dataset shows the average rating of those that feel satisfied about their lives (where 0 is 'not at all satisfied' and 10 is 'completely satisfied'), for adults aged 16 years and over, in the UK, for periods 04/2011-03/2012 to 04/2022-03/2023.",
+        "publisher": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+        "creator": "https://staging.idpd.uk/publishers/department-for-education",
```

Comment:
Please change to `"https://staging.idpd.uk/publishers/office-for-national-statistics"`

A good reference is the published dataset on PMD-Beta